modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -6,3 +6,20 @@
 		"Brig Physician",
 		"Combat Medic",
 	)
+
+/datum/job/assistant
+	alt_titles = list(
+		"Assistant",
+		"Artist",
+		"Barber",
+		"Businessman",
+		"Businesswoman",
+		"Civilian",
+		"Entertainer",
+		"Freelancer",
+		"Tourist",
+		"Trader",
+		"Off-Duty Crew",
+		"Off-Duty Staff",
+		"Musician",
+	)
```

Comment:
I feel like this should definitely be a modular addition to that list in New()- or simply a base code edit.

Reply:
Gonna give that a try in a bit then resolve this :>